packages/components/collapsible-panel/src/collapsible-panel.spec.js

```diff
@@ -160,85 +160,62 @@ it('should forward data-attributes', () => {
   expect(container.querySelector("[data-foo='bar']")).toBeInTheDocument();
 });
 
-describe('getPanelContentId', () => {
-  it('should return a string containing the given id', () => {
-    const panelContentId = CollapsiblePanel.getPanelContentId('example');
-
-    expect(panelContentId).toEqual(expect.stringContaining('example'));
-  });
-});
-
 describe('aria attributes', () => {
   const renderPanel = (props) => {
     const { container } = render(
-      <CollapsiblePanel
-        id="example"
-        header="Header"
-        onToggle={jest.fn()}
-        {...props}
-      >
+      <CollapsiblePanel header="Header" onToggle={jest.fn()} {...props}>
         Children
       </CollapsiblePanel>
     );
-    const getPanelHeader = () => screen.getByRole('button');
 
-    const panelContentId = CollapsiblePanel.getPanelContentId(
-      props.id || 'example'
+    const panelHeader = screen.getByRole('button');
+    const panelHeaderId = panelHeader.getAttribute('id');
+    const panelContent = container.querySelector(
+      `[aria-labelledby="${panelHeaderId}"]`
     );
-    const getPanelContent = () =>
-      container.querySelector(`[id=${panelContentId}]`);
 
     return {
-      getPanelHeader,
-      getPanelContent,
+      panelHeader,
+      panelContent,
     };
   };
   it('should have a valid aria-controls correspondence', () => {
-    const { getPanelHeader, getPanelContent } = renderPanel({
+    const { panelHeader, panelContent } = renderPanel({
       id: 'test-id',
       isClosed: false,
     });
 
-    const panelContentId = CollapsiblePanel.getPanelContentId('test-id');
-
-    // assert that the header button has the aria attribute
-    const headerButton = getPanelHeader();
-    expect(headerButton).toHaveAttribute('aria-controls', panelContentId);
```

Comment:
This is now implicitly done in the `renderPanel` function to get access to the panel content element.